cmd_set.go

```diff
@@ -660,29 +667,38 @@ func (m *Miniredis) cmdSscan(c *server.Peer, cmd string, args []string) {
 	withTx(m, c, func(c *server.Peer, ctx *connCtx) {
 		db := m.db(ctx.selectedDB)
 		// return _all_ (matched) keys every time
-
-		if opts.cursor != 0 {
```

Comment:
Should we validate that cursor is `>0`?

Reply:
I moved this validation up, and is already done in line 637 to line 646. It checks to make sure its not less then 0 or equal to 0.